--- ruby-netcdf-0.5.2/extconf.rb~	2003-08-12 23:24:39.000000000 +0900
+++ ruby-netcdf-0.5.2/extconf.rb	2003-09-16 18:08:15.575707200 +0900
@@ -31,8 +31,8 @@
    exit(-1)
 end
 
-if defined?(__CYGWIN__) && VERSION < '1.7'
-   have_library("narray") || raise("ERROR: narray brary is not found")
+if /cygwin|mingw/ =~ RUBY_PLATFORM
+   have_library("narray") || raise("ERROR: narray library is not found")
 end
 
 create_makefile "numru/netcdfraw"
