diff -urN netcdf-3.5.0_orig/src/cxx/ncvalues.cpp netcdf-3.5.0/src/cxx/ncvalues.cpp
--- netcdf-3.5.0_orig/src/cxx/ncvalues.cpp	1998-12-23 03:21:49.000000000 +0900
+++ netcdf-3.5.0/src/cxx/ncvalues.cpp	2003-09-15 22:05:57.503014400 +0900
@@ -304,7 +304,7 @@
 
 ostream& NcValues_float::print(ostream& os) const
 {
-    long save=os.flags();
+    std::_Ios_Fmtflags save=os.flags();
     os.precision(7);
     for(int i = 0; i < the_number - 1; i++)
       os << the_values[i] << ", ";
@@ -316,7 +316,7 @@
 
 ostream& NcValues_double::print(ostream& os) const
 {
-    long save=os.flags();
+    std::_Ios_Fmtflags save=os.flags();
     os.precision(15);
     for(int i = 0; i < the_number - 1; i++)
       os << the_values[i] << ", ";
diff -urN netcdf-3.5.0_orig/src/cxx/netcdfcpp.h netcdf-3.5.0/src/cxx/netcdfcpp.h
--- netcdf-3.5.0_orig/src/cxx/netcdfcpp.h	2001-02-06 05:29:12.000000000 +0900
+++ netcdf-3.5.0/src/cxx/netcdfcpp.h	2003-09-15 22:05:57.583129600 +0900
@@ -154,7 +154,7 @@
     virtual ~NcDim( void );
     
     // to construct dimensions, since constructor is private
-    friend NcFile;
+    friend class NcFile;
 };
 
 
@@ -357,7 +357,7 @@
     void init_cur( void );
 
     // to make variables, since constructor is private
-  friend NcFile;
+  friend class NcFile;
 };
 
 
@@ -388,7 +388,7 @@
     NcAtt( NcFile*, NcToken); // global attribute
     
     // To make attributes, since constructor is private
-  friend NcFile;
+  friend class NcFile;
   friend NcAtt* NcVar::get_att( NcToken ) const;
 };
 
diff -urN netcdf-3.5.0_orig/src/ncdump/Makefile netcdf-3.5.0/src/ncdump/Makefile
--- netcdf-3.5.0_orig/src/ncdump/Makefile	2001-03-14 02:42:52.000000000 +0900
+++ netcdf-3.5.0/src/ncdump/Makefile	2003-09-15 22:06:15.468848000 +0900
@@ -10,7 +10,7 @@
 c_sources	= ncdump.c vardata.c dumplib.c
 headers		= ncdump.h vardata.h dumplib.h
 
-PROGRAM		= ncdump
+PROGRAM		= ncdump.exe
 
 PACKING_LIST	= $(c_sources) $(headers) depend \
 		  test0.cdl ncdump.1 Makefile
diff -urN netcdf-3.5.0_orig/src/ncgen/Makefile netcdf-3.5.0/src/ncgen/Makefile
--- netcdf-3.5.0_orig/src/ncgen/Makefile	2001-03-14 02:42:53.000000000 +0900
+++ netcdf-3.5.0/src/ncgen/Makefile	2003-09-15 22:06:26.945350400 +0900
@@ -5,7 +5,7 @@
 include ../macros.make
 
 NCDUMP		= ../ncdump/ncdump
-PROGRAM		= ncgen
+PROGRAM		= ncgen.exe
 INCLUDES	= -I../libsrc -I.
 MANUAL		= ncgen.1
 GARBAGE		= $(PROGRAM) \
