*** /etc/bash.bashrc.orig	2009-11-09 14:08:16.000000000 +0900
--- /etc/bash.bashrc	2009-11-09 14:08:20.000000000 +0900
***************
*** 45,47 ****
--- 45,54 ----
  		fi
  	}
  fi
+ 
+ # RubyGems
+ GEM_HOME=/var/lib/gems/1.8; export GEM_HOME
+ export PATH=/var/lib/gems/1.8/bin:$PATH
+ export RUBYLIB=/var/lib/gems/1.8/lib:$RUBYLIB
+ # disable G95 Warnings
+ G95_MEM_SEGMENTS=0; export G95_MEM_SEGMENTS
