*** chroot/etc/dhcp3/dhclient.conf.orig	2009-04-12 23:03:36.000000000 +0900
--- chroot/etc/dhcp3/dhclient.conf	2009-04-12 23:05:05.000000000 +0900
***************
*** 23,30 ****
  	netbios-name-servers, netbios-scope, interface-mtu,
  	rfc3442-classless-static-routes;
  #require subnet-mask, domain-name-servers;
! #timeout 60;
! #retry 60;
  #reboot 10;
  #select-timeout 5;
  #initial-interval 2;
--- 23,30 ----
  	netbios-name-servers, netbios-scope, interface-mtu,
  	rfc3442-classless-static-routes;
  #require subnet-mask, domain-name-servers;
! timeout 1;
! retry 0;
  #reboot 10;
  #select-timeout 5;
  #initial-interval 2;
