*** chroot/usr/share/pyshared/GDebi/GDebi.py.orig	2009-06-11 09:45:33.000000000 +0900
--- chroot/usr/share/pyshared/GDebi/GDebi.py	2009-06-11 09:45:33.000000000 +0900
***************
*** 355,361 ****
              os.execl("/usr/bin/gksu", "gksu", "--desktop",
                       "/usr/share/applications/gdebi.desktop",
                       "--message","<big><b>%s</b></big>\n\n%s" % (msg_hdr,msg_bdy),
-                      "--always-ask-pass",
                       "--", "gdebi-gtk", "--non-interactive",
                       self._deb.file)
  
--- 355,360 ----
