*** chroot/etc/mozpluggerrc.orig	2009-06-09 10:57:52.000000000 +0900
--- chroot/etc/mozpluggerrc	2009-06-09 10:57:52.000000000 +0900
***************
*** 185,191 ****
  audio/x-mid:midi,mid:MIDI audio file
  audio/midi:midi,mid:MIDI audio file
  audio/x-midi:midi,mid:MIDI audio file
! 	controls noisy stream: timidity -Od "$file"
  	controls: playmidi "$file"
  
  audio/mod:mod:Soundracker audio Module
--- 185,191 ----
  audio/x-mid:midi,mid:MIDI audio file
  audio/midi:midi,mid:MIDI audio file
  audio/x-midi:midi,mid:MIDI audio file
! 	controls noisy stream: timidity -Os "$file"
  	controls: playmidi "$file"
  
  audio/mod:mod:Soundracker audio Module
***************
*** 222,227 ****
--- 222,228 ----
  audio/x-mpeg-url:m3u:MPEG music resource locator
  audio/x-scpls:pls:Shoutcast Playlists
  #	controls: mpg321 -q -@ "$file"
+ 	controls: totem "$file"
  	HELPER(xmms -e -p "$file")
  
  audio/x-ogg:ogg:OGG audio
***************
*** 315,320 ****
--- 316,322 ----
  text/x-pdf:pdf:PDF file
  	ACROREAD()
  	repeat noisy swallow(evince) fill: evince "$file"
+ 	repeat noisy swallow(epdfview) fill: epdfview "$file"
  	repeat noisy swallow(kpdf) fill: kpdf "$file"
  	repeat noisy swallow(Xpdf) fill: xpdf -g +9000+9000 "$file"
  	GV()
