*** chroot/usr/share/hal/fdi/policy/10osvendor/20-storage-methods.fdi.orig	2009-01-08 07:42:51.000000000 +0900
--- chroot/usr/share/hal/fdi/policy/10osvendor/20-storage-methods.fdi	2009-09-30 17:22:07.000000000 +0900
***************
*** 246,251 ****
--- 246,252 ----
  	  <append key="volume.mount.valid_options" type="strlist">uid=</append>
  	  <append key="volume.mount.valid_options" type="strlist">gid=</append>
  	  <append key="volume.mount.valid_options" type="strlist">umask=</append>
+ 	  <append key="volume.mount.valid_options" type="strlist">locale=</append>
  	  <append key="volume.mount.valid_options" type="strlist">utf8</append>
  	</match>
  	<match key="/org/freedesktop/Hal/devices/computer:system.kernel.name" string="FreeBSD">
